openpype/modules/deadline/plugins/publish/submit_max_deadline.py

```diff
@@ -216,3 +249,40 @@ def _clean_name(path):
         plugin_info.update(plugin_data)
 
         return job_info, plugin_info
+
+    @classmethod
+    def get_attribute_defs(cls):
+        defs = super(MaxSubmitDeadline, cls).get_attribute_defs()
+        defs.extend([
+            BoolDef("use_published",
+                    default=cls.use_published,
+                    label="Use Published Scene"),
+
+            NumberDef("priority",
+                      minimum=1,
+                      maximum=250,
+                      decimals=0,
+                      default=cls.priority,
+                      label="Priority"),
+
+            NumberDef("chunkSize",
+                      minimum=1,
+                      maximum=50,
+                      decimals=0,
+                      default=cls.chunkSize,
+                      label="Frame Per Task"),
+
+            TextDef("group",
+                    default=cls.group,
+                    label="Group Name"),
+
+            TextDef("deadline_pool",
+                    default=cls.deadline_pool,
+                    label="Deadline Pool"),
+
+            TextDef("deadline_pool_secondary",
+                    default=cls.deadline_pool_secondary,
+                    label="Deadline Pool Secondary")
```

Comment:
I would avoid adding a `deadline` prefix. The attributes are stored as Plug-in Attributes on the instance per plug-in. So the attribute on the instance would already be e.g. `publish_attributes.MaxSubmitDeadline.pool` as far as I know.